README.md

```diff
@@ -28,7 +28,7 @@ a dockerized Espresso Sequencer network with an example Layer 2 rollup applicati
 ## Run the tests
 
     just pull # to pull docker images
-    cargo test --all-features
```

Comment:
Indeed it is needed to use "--all-features" to run the integration tests. However it is even better to run
```
just test
```